src/modules/signup/modal-additional-info/ModalAdditionalInfo.vue

```diff
@@ -97,7 +97,7 @@
             class="w-full h-3.5"
             :isError="!isValidPhone"
           />
-          <p v-if="!isValidPhone" class="text-sm text-red-5 mt-0.25 ml-1.5">
+          <p v-show="!isValidPhone" class="text-sm text-red-5 mt-0.25 ml-1.5">

```

Comment:
This error can be made into a single error component, and can be reusable in `src\modules\signup\modal-additional-info\ModalAdditionalInfo.vue`

Reply:
Already created a new component for it. But did not change because out of this branch scope